test/e2e/capi_test.go

```diff
@@ -265,14 +242,15 @@ var _ = Describe("Running the Cluster API E2E tests", func() {
 						InitWithBootstrapProviders:      []string{"kubeadm:" + e2eConfig.GetVariable(OldCAPIUpgradeVersion)},
 						InitWithControlPlaneProviders:   []string{"kubeadm:" + e2eConfig.GetVariable(OldCAPIUpgradeVersion)},
 						InitWithInfrastructureProviders: []string{"azure:" + e2eConfig.GetVariable(OldProviderUpgradeVersion)},
+						InitWithAddonProviders:          []string{"azure:" + "v0.1.1-alpha.0"},
```

Comment:
Can this be set to an environment variable in `azure-dev.yaml` ?

Reply:
Sure, I just wanted to verify that it would actually work first.